spec.html

```diff
@@ -8871,18 +8871,24 @@ <h1>[[Construct]] ( _argumentsList_, _newTarget_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-createbuiltinfunction" aoid="CreateBuiltinFunction">
-      <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _prototype_ ] ] )</h1>
-      <p>The abstract operation CreateBuiltinFunction takes arguments _steps_ and _internalSlotsList_ (a List of names of internal slots) and optional arguments _realm_ and _prototype_. _internalSlotsList_ contains the names of additional internal slots that must be defined as part of the object. This operation creates a built-in function object. It performs the following steps when called:</p>
+      <h1>CreateBuiltinFunction ( _steps_ [ , _name_ [ , _length_ [ , _additionalInternalSlotsList_ [ , _realm_ [ , _prototype_ ] ] ] ] ] )</h1>
```

Comment:
I'd prefer to always be explicit about the name and length, rather than introducing new optional parameters.

Reply:
```suggestion
      <h1>CreateBuiltinFunction ( _steps_, _name_, _length_, _additionalInternalSlotsList_ [ , _realm_ [ , _prototype_ ] ] ] ] ] )</h1>
```